src/server.c

```diff
@@ -397,37 +397,36 @@ server_init(struct server *server)
 		wlr_xwayland_create(server->wl_display, compositor, true);
 	if (!server->xwayland) {
```

Comment:
Could be negated, thus the `wlr_log` and `exit` being removed and the `if` changed to
```c
if (server->xwayland) {
  ..
}
```